composer.json

```diff
@@ -1,5 +1,26 @@
 {
+    "name": "marmelab/dobble-php",
+    "license": "MIT",
+    "description": "Command-line card generator for dobble-like game",
+    "authors": [
+        {
+            "name": "kmaschta",
+            "email": "kevin@marmelab.com"
+        }
+    ],
+    "autoload": {
+        "psr-4": {"": "src/"}
+    },
     "require": {
         "symfony/console": "^2.7"
+    },
+    "autoload-dev": {
+        "psr-4": {"": "tests/"}
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.0"
+    },
+    "scripts": {
+        "test": ["phpunit tests/DobbleCommandTest"]
```

Comment:
Have a look on [phpunix.xml](https://github.com/symfony/symfony-standard/blob/2.8/app/phpunit.xml.dist) to test directly the whole `tests` folder.